project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,206 @@
+
+upgrades:
+  - upgrade_name: All ENERGYSTAR Appliances
+    options:
+      - &energystar_refrigerator
+        option: Refrigerator|EF 19.9
+        apply_logic:
+          - or:
+            - Refrigerator|EF 6.7
+            - Refrigerator|EF 10.2
+            - Refrigerator|EF 10.5
+            - Refrigerator|EF 15.9
+            - Refrigerator|EF 17.6
+        costs:
+          - value: 2500.0 #TBD
+            multiplier: Fixed (1)
+        lifetime: 17.4
+      - &energystar_clothes_washer
+        option: Clothes Washer|EnergyStar
+        apply_logic:
+          - Clothes Washer|Standard
+      - &energystar_dishwasher 
+        option: Dishwasher|270 Rated kWh
+        apply_logic:
+          - or:
+            - Dishwasher|318 Rated kWh
+            - Dishwasher|290 Rated kWh
+      - &energystar_clothes_dryer_electric 
+        option: Clothes Dryer|Electric, Premium, EnergyStar
+        apply_logic:
+          - or:
+            - Clothes Dryer|Electric
+            - Clothes Dryer|Electric, Premium
+      - &energystar_clothes_dryer_gas 
+        option: Clothes Dryer|Gas, Premium, EnergyStar
+        apply_logic:
+          - or:
+            - Clothes Dryer|Gas
+            - Clothes Dryer|Gas, Premium
+
+  - upgrade_name: All ENERGYSTAR Most Efficient Appliances
+    options:
+      - &energystar_most_eff_refrigerator
+        option: Refrigerator|EF 15.9 #TBD
+        apply_logic:
+          - or:
+            - Refrigerator|EF 6.7
+            - Refrigerator|EF 10.2
+            - Refrigerator|EF 10.5
+        costs:
+          - value: 2500.0 #TBD
+            multiplier: Fixed (1)
+        lifetime: 17.4
+      - &energystar_most_eff_clothes_washer
+        option: Clothes Washer|EnergyStar Most Efficient
+        apply_logic:
+        - or:
+            - Clothes Washer|Standard
+            - Clothes Washer|EnergyStar
+            - Clothes Washer|EnergyStar, Cold Only
+            - Clothes Washer|EnergyStar More Efficient
+      - &energystar_most_eff_dishwasher 
+        option: Dishwasher|240 Rated kWh
+        apply_logic:
+          - or:
+            - Dishwasher|318 Rated kWh
+            - Dishwasher|290 Rated kWh
+            - Dishwasher|270 Rated kWh
+            - Dishwasher|255 Rated kWh
+      - &energystar_most_eff_clothes_dryer_electric 
+        option: Clothes Dryer|Electric, Premium, EnergyStar Most Efficient
+        apply_logic:
+          - or:
+            - Clothes Dryer|Electric
+            - Clothes Dryer|Electric, Premium
+            - Clothes Dryer|Electric, Premium, EnergyStar
+      - &energystar_most_eff_clothes_dryer_gas 
+        option: Clothes Dryer|Gas, Premium, EnergyStar Most Efficient
+        apply_logic:
+          - or:
+            - Clothes Dryer|Gas
+            - Clothes Dryer|Gas, Premium
+            - Clothes Dryer|Gas, Premium, EnergyStar
+
+  - upgrade_name: ENERGYSTAR Electric Dryer
```

Comment:
If this is for non-electric replacement only, consider making upgrade name more explicit

Reply:
done